### PR TITLE
Environment setup

### PR DESCRIPTION
**Pull request by NegativeKarma**
## What we have learned
- We learned to start setting up the test environment
- For now we didn't have problems or blockers

### CODE REVIEW

**Comment by NegativeKarma** [comment]
this is the link to the specific chore https://www.pivotaltracker.com/n/projects/2132546/stories/153219878

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  # The different available types are documented in the features, such as in
+  # https://relishapp.com/rspec/rspec-rails/docs
+  config.infer_spec_type_from_file_location!
+
+  # Filter lines from Rails gems in backtraces.
+  config.filter_rails_from_backtrace!
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
+end
+
```
Comment (tochman) [inline spec/rails_helper.rb]: Please clean this from comments

File: features/support/env.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+#
+#   Before('~@no-txn', '~@selenium', '~@culerity', '~@celerity', '~@javascript') do
+#     DatabaseCleaner.strategy = :transaction
+#   end
+#
+
+# Possible values are :truncation and :transaction
+# The :transaction strategy is faster, but might give you threading problems.
+# See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
+Cucumber::Rails::Database.javascript_strategy = :truncation
```
Comment (tochman) [inline features/support/env.rb]: Please clean up this from comments

**Review by tochman — DISMISSED** [review]
Please remove `coverage` from version control

File: features/support/env.rb (hunk 2 above)
Comment (NegativeKarma) [reply features/support/env.rb]: Done

File: spec/rails_helper.rb (hunk 1 above)
Comment (NegativeKarma) [reply spec/rails_helper.rb]: Done

**Review by diraulo — APPROVED** [review]
LGTM :shipit:

**Pull request merged**